helmfile/charts/prometheus-alerts/values.yaml

```diff
@@ -62,6 +62,7 @@ defaultRules:
     kubeProxy: true
     kubeControllerManager: true
     configReloaders: true
+    networkpolicies: false
```

Comment:
It would be great if we had a way to enable and disable them via the config.

Reply:
sure, I'll look into that and see if there is a way

Reply:
Not really, I didn't manage to find a way to do that through the config. 
Do you have any ideas that I can try ?

Reply:
You should be able to do it like we have for some here https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/prometheus-alerts-sc.yaml.gotmpl#L19

Reply:
It's updated now to read from the config